src/desktop/game_mode.rs

```diff
@@ -113,13 +113,13 @@ impl<'a> GameMode<'a> {
     #[doc(alias = "QueryStatusByPIDFd")]
     pub async fn query_status_by_pidfd(
         &self,
-        target: &BorrowedFd<'_>,
-        requester: &BorrowedFd<'_>,
+        target: impl AsFd,
+        requester: impl AsFd,
```

Comment:
Why not take &impl AsFd instead? If you will just end up borrowing

Reply:
Changed it to `&impl AsFd`.